metisp/pymetis/src/pymetis/recipes/instrument/metis_pupil_imaging.py

```diff
@@ -166,12 +166,30 @@ class MetisPupilImaging(MetisRecipe):
     _email = "jkarr@asiaa.sinica.edu.tw"
     _copyright = "GPL-3.0-or-later"
     _synopsis = "Basic processing of pupil images"
-    _description = (
-            "This recipe performs basic reduction (dark current subtraction, flat fielding,\n"
-            "optional bias subtraction, persistance and linearity corrections) on engineering\n"
-            "images of the pupil masks. This recipe is not expected to be used by observers\n"
-            "during regular use."
-    )
+    _description = """
+       This recipe performs basic reduction (dark current subtraction, flat fielding,\n
+       optional bias subtraction, persistance and linearity corrections) on engineering\n
+       images of the pupil masks. This recipe is not expected to be used by observers\n
```

Comment:
Related, you don't need these `\n` here, because the triple quotes already take the actual line breaks into account. So printing this string will have double line breaks.

Reply:
I can add a test for that ;-)